inst/make_hexsticker.R

```diff
@@ -86,6 +116,30 @@ covid_map_2 <- ggplot() +
 
 print(covid_map_2)
 
+covid_map_3 <- ggplot() +
+  ggspatial::layer_spatial(data = world_without_regions, size = 0.01) +
+  coord_sf(crs = "ESRI:54016") +
+  # scale_fill_manual(
+  #  name = "",
+  #  values = c("#0072b2", "#cc79a7", "grey80")
+  # ) +
+  # scale_color_manual(
+  #   name = "",
+  #   values = c("black", "black", "#666666")
+  # ) +
+  # scale_size_manual(
+  #   name = "",
+  #   values = c(0.1, 0.1, 0.018)
+  # ) +
```

Comment:
With some meddling I've managed to put borders back in but they're currently red and I've not figured out why.

The polygons provided by the `ne_states` call don't know which of their edges are "external" so there's no straightforward way to draw the outlines of the countries using this geometry.

We *could* try drawing outlines just on some of the countries using the underlying system, and this may be a simpler approach.

The current approach is to draw all countries, then draw the supported countries with a wide outline (I have not figured out how to have an empty fill, sorry), then draw the regions of the supported countries.

Currently the outline of the country is reddish pink and I've not managed to fix that either. I've made some progress but will have to stop for tonight.

Reply:
That is wierd. I'm not a fan of the orange borders, ideally these should be black and thinner but if its to much of a hassle I'm happy with it without borders

Reply:
shall we merge into the main hex branch and try and resolve the final tweak there? I agree this is great. If I get the chance I will bang my head into the outline issue as it does seem strange.

Reply:
I've done a slight reordering.

We now convert the countries outlines into lines (instead of polygons) and draw them *after* we draw the coloured regions. This means that we now draw borders between countries for which we have regional data.

I know we didn't like the orange, so is it ... better? ... that now the border appears to be blue by default? I think there is a simple `ggplot` scale/colour error I am making and I hope someone with more expertise may figure it out, though the `spatial_layer` objects are particular in how they approach aesthetics.

In any case, I think this is closer and would be happy if this were merged into the mainstream hexsticker branch (and possibly tidied to reduce the proliferation and numbering glitches of options) and then polishing can be done there.

In terms of things which could be done to make it cleaner, `ne_countries` and `ne_states` appear to have different _default_ scales of the geometries they return, which is why the code simplifies the country outlines down to 4% of original points and the regions only to 50% of original points. Discrepancies only seem to be visible when very closely zoomed in (where regions are drawn over countries with slightly different outlines) but it should be possible to specify the same scale for both and then simplify them by the same factor.